### PR TITLE
add more printer profiles

### PR DESCRIPTION
**Pull request by fr3ak2402**
Changes:
Removal of the Artillery Sidewinder X1 Printing Plates Texture, as it is redundant and has been replaced with a new version.

New printers:

Artillery Sidewinder X1:
Printer image (180x180)
Custom Printing Plate Texture (512x512)(SVG) - the manufacturer plate imitated
0.4 Nozzle Profile
0.15 Process Profile (Optimal)
0.20 Process Profile (Standard)
0.25 Process Profile (Draft)

Elegoo Neptune:
Printer image (180x180)
Custom Printing Plate Texture (PNG) - the manufacturer plate imitated
0.4 Nozzle Profile
0.08 Process Profile (Extra Fine)
0.12 Process Profile (Fine)
0.16 Process Profile (Optimal)
0.20 Process Profile (Standard)
0.24 Process Profile (Draft)
0.28 Process Profile (Extra Draft)

### CODE REVIEW

**Review by SoftFever — CHANGES_REQUESTED** [review]

File: resources/profiles/Elegoo/machine/Elegoo Neptune 2S 0.4 nozzle.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
...
+	"machine_start_gcode": "M413 S0 ; disable Power Loss Recovery\nG90 ; use absolute coordinates\nM83 ; extruder relative mode\nM104 S120 ; set temporary nozzle temp to prevent oozing during homing and auto bed leveling\nM140 S[hot_plate_temp_initial_layer] ; set final bed temp\nG4 S10 ; allow partial nozzle warmup\nG28 ; home all axis\n;G29 ; run abl mesh\nM420 S1 ; load mesh\nG1 Z50 F240\nG1 X2 Y10 F3000\nM104 S[nozzle_temperature_initial_layer] ; set final nozzle temp\nM190 S[hot_plate_temp_initial_layer] ; wait for bed temp to stabilize\nM109 S[nozzle_temperature_initial_layer] ; wait for nozzle temp to stabilize\nG1 Z0.28 F240\nG92 E0\nG1 Y140 E10 F1500 ; prime the nozzle\nG1 X2.3 F5000\nG92 E0\nG1 Y10 E10 F1200 ; prime the nozzle\nG92 E0",
```
[the remark this answers is not in the document]
Comment (fr3ak2402) [reply resources/profiles/Elegoo/machine/Elegoo Neptune 2S 0.4 nozzle.json:109]: Yes wait have made a mistake, when replacing and adding the printers

[the remark this answers is not in the document]
Comment (fr3ak2402) [reply resources/profiles/Elegoo/machine/Elegoo Neptune 2S 0.4 nozzle.json]: @SoftFever 
So I have adjusted it, I must have made a mistake earlier when copying the files

[the remark this answers is not in the document]
Comment (fr3ak2402) [reply resources/profiles/Elegoo/machine/Elegoo Neptune 2S 0.4 nozzle.json]: @SoftFever
now it should fit again everything with the start code parameters

**Review by SoftFever — APPROVED** [review]

**Comment by SoftFever** [comment]
Looks good.
You're the man!

**Pull request merged**